supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -8,6 +8,21 @@ Follow the _IBM Style_ guide advice of _day Month year_ for date formats, for ex
 
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
+[[man-pages]]
+== Man page references
+
+When referencing a man page in an additional resources section, use the following format: `_<man_page_name>(<section_number>)_` _man page on your system_.
```

Comment:
```suggestion
When referencing a man page in an "Additional resources" section, use the following format: 
* `_<man_page_name>(<section_number>)_` man page on your system
```

Reply:
No sorry, I missed this suggestion! I agree that this looks better, will update.